test/powershell/Language/Operators/TernaryOperator.Tests.ps1

```diff
@@ -0,0 +1,185 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+using namespace System.Management.Automation
+using namespace System.Management.Automation.Language
+
+Describe "Ternary Operator" -Tags CI {
+    Context "Parsing of ternary operator" {
+        BeforeAll {
+            $testCases_1 = @(
+                @{ Script = '$true?2:3'; TokenKind = [TokenKind]::Variable; }
```

Comment:
Looks like we are space-sensitive :-( - this is area for typo errors in scripts.

Reply:
Well, this is the existing behavior that we cannot break.